rollbar.go

```diff
@@ -0,0 +1,82 @@
+package rollbar
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const apiBaseUrl string = "https://api.rollbar.com/api/1/"
+
+var apiKey string
```

Comment:
The `apiKey` should not be optional because there is no default apiKey and every client should have it's own api Key.